ipaserver/install/server/install.py

```diff
@@ -941,6 +933,18 @@ def install(installer):
     except Exception:
         raise ScriptError("Configuration of client side components failed!")
 
+    # Enable configured services and update DNS SRV records
+    service.enable_services(host_name)
+    api.Command.dns_update_system_records()
+    if not options.setup_dns:
+        # Create a BIND instance
+        bind = bindinstance.BindInstance(fstore)
+        bind.setup(host_name, ip_addresses, realm_name,
+                   domain_name, (), 'first', (),
+                   zonemgr=options.zonemgr,
+                   no_dnssec_validation=options.no_dnssec_validation)
+        bind.create_file_with_system_records()
+
```

Comment:
I think we need @abbra opinion on this. I'm guessing that IPA DNS needs to be configured before AD trust.

Reply:
``create_file_with_system_records`` just dumps a file to file system.

Reply:
It is the bind.setup() that I'm most concerned about.

Reply:
@rcritten yes, if IPA DNS is present, adtrust install will use it to set up DNS entries for `_msdcs`.